data/authors.yaml

```diff
@@ -17,3 +17,5 @@ authors:
       linkedInLink: https://www.linkedin.com/in/shailishah5632/
     - slug: caleb-hock
       name: Caleb Hock
+    - slug: abhinav-sharma
```

Comment:
@Abhinav-CHD — would you like to add a bio, Twitter handle or LinkedIn link to your author data? It would look similar to [this author profile page](https://learn.sourcegraph.com/authors/marek-zaluski). If so, feel free to commit it.

Reply:
Hi, I have updated authors.yaml file